c2corg_api/models/cache_version.py

```diff
@@ -372,7 +372,7 @@ def update_cache_version_associations(
                 association['child_type'] == OUTING_TYPE:
             routes_to_update.add(association['parent_id'])
 
-    if ignore_document_id is not None:
+    if ignore_document_id is not None and len(documents_to_update):
```

Comment:
It's considered more *pythonic* to write:
```
if my_list:
    ...
```
instead of:
```
if len(my_list):
    ...
```
or
```
if len(my_list) > 0:
    ...
```

See also: https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations